### PR TITLE
V6/fix swap current balance

### PR DESCRIPTION
**Pull request by hsurf22**
### Devop

* \[x] Updated CHANGELOG.md
* \[x] Add PR label
   - Add top right text component
   - Fix swap current balance

Screenshot from 2021-05-28 15-05-48

Screenshot from 2021-05-28 15-04-56

### CODE REVIEW

File: src/views/components-wallet/TheWalletTopRightText.vue [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<template>
```
Comment (olgakup) [inline src/views/components-wallet/TheWalletTopRightText.vue:1]: Please implement the following: 
- move the component to core/components
- rename the component to AppButtonBalance


views folder is disignanted to layouts and components that are specific to the layout.  please refer to the vue guide on how to name components

File: src/views/components-wallet/TheWalletTopRightText.vue [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  </div>
+</template>
+
+<script>
+export default {
+  components: {},
+  props: {
+    loading: {
+      type: Boolean,
+      default: false
```
Comment (olgakup) [inline src/views/components-wallet/TheWalletTopRightText.vue:14]: change default loading to  true

File: src/views/components-wallet/TheWalletTopRightText.vue [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    text: {
```
Comment (olgakup) [inline src/views/components-wallet/TheWalletTopRightText.vue:16]: - rename text prop, to balance
- implement computed  texts as: 
return string  --> `'${$t('balance')}: ${this.balance}'` . Where $t('balance') is the balance string to translate, and this.balance is a balance prop

File: src/modules/swap/ModuleSwap.vue [hunk 4]
```diff
@@ -20,21 +20,22 @@
...
             <v-col cols="12" sm="5" class="pb-0 pb-sm-3">
-              <mew-select
-                :value="fromTokenType"
-                label="From"
-                :items="actualFromTokens"
-                :is-swap="true"
-                :loading="isLoading"
-                @input="setFromToken" />
+              <div class="position--relative">
+                <wallet-top-right-text
```
Comment (olgakup) [inline src/modules/swap/ModuleSwap.vue:26]: - mew-component--top-right-text class makes the element `right: 8px`; of right to the component, In the design the element is positioned as` right: 0px`
- change color to green

**Review by olgakup — CHANGES_REQUESTED** [review]

**Comment by olgakup** [comment]
Please in the future do not mix features when making a pr. ENS icons could have been based of the develop branch and merged sooner

File: src/views/components-wallet/TheWalletTopRightText.vue (hunk 3 above)
Comment (hsurf22) [reply src/views/components-wallet/TheWalletTopRightText.vue:16]: fixed

File: src/modules/swap/ModuleSwap.vue (hunk 4 above)
Comment (hsurf22) [reply src/modules/swap/ModuleSwap.vue:26]: fixed

**Pull request merged**